#!/usr/bin/env python

PACKAGE = "fer_rari"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("speed", double_t, 0, "Race car speed", 2.5, 0, 20)
gen.add("coefficient", double_t, 0, "Coefficient k in equation ld = k*v", 0.2, 0.1, 1)
gen.add("decelerate_dist", double_t, 0, "Distance from the goal location at which robot starts decreasing speed", 0.2, 1, 5)

exit(gen.generate(PACKAGE, "Pure_pursuit_node", "Racecar"))
